InvenTree/plugin/samples/integration/locate_sample.py

```diff
@@ -6,18 +6,22 @@
 
 import logging
```

Comment:
I would recommend moving this to InvenTree/plugin/samples/locate/locate_sample.py